libraries/wasm-jit/Include/IR/Operators.h

```diff
@@ -14,60 +14,60 @@ namespace IR
 
 	struct ControlStructureImm
 	{
-		ResultType resultType;
+		ResultType resultType{};
```

Comment:
In this file valgrind found only issue with uninitialized resultType.
However I initialized by default all other structures just in case.

Stack trace for this issue:
```
==12171==    at 0x2B02083: void WASM::serialize<Serialization::MemoryInputStream>(Serialization::MemoryInputStream&, IR::ControlStructureImm&, IR::FunctionDef const&) (WASMSerialization.cpp:234)
==12171==    by 0x2AE5E1F: WASM::serializeFunctionBody(Serialization::InputStream&, IR::Module&, IR::FunctionDef&) (WASMSerialization.cpp:531)
```

resultType is being checked in WASMSerialization.cpp:234:
`I8 encodedResultType = imm.resultType == ResultType::none ? -64 : -(I8)imm.resultType;`
However the argument `ControlStructureImm& imm` is being passed up from function above: WASMSerialization.cpp:531.
The declaration of `ConstrolStructureImm& imm` I believe is in WASMSerialization.cpp:525 - which (as a good practice) could also be initialized.
However fixing the issue at the very bottom in file Operators.h will work for any usage of ControlStructureImm.

Above scenario happens for every use of `WASM::serializeFunctionBody`. One example is unit test:
`plugins/chain_plugin/test/test_account_query_db`

Reply:
There might be a slight performance cost if we initialize structs needlessly (if they are being overwritten in all cases). Is this a tradeoff we want?

Reply:
I don't think this is performance critical code.